### PR TITLE
[FE-5318] ensure float precision matches mapdql/core

### PR DESCRIPTION
**Pull request by arittr**
Ensure float-type numbers use the same precision/rounding as core (i.e. `std::numeric_limits<float>::digits10 + 1`).

### CODE REVIEW

File: dist/node-connector.js [hunk 1]
```diff
@@ -18913,98 +18864,115 @@ module.exports =
...
-	    if (
-	      listeners.fn === fn &&
-	      (!once || listeners.once) &&
-	      (!context || listeners.context === context)
-	    ) {
-	      clearEvent(this, evt);
-	    }
-	  } else {
-	    for (var i = 0, events = [], length = listeners.length; i < length; i++) {
+	  if (fn) {
```
Comment (uyanga-gb) [inline dist/node-connector.js]: Can we combine these three if statements?

File: src/process-columnar-results.js [hunk 2]
```diff
@@ -75,6 +75,12 @@ export default function processColumnarResults(
               )
               break
             case "FLOAT":
+              const value = data.columns[c].data.arr_col[r].data.real_col[e]
```
Comment (jrajav) [inline src/process-columnar-results.js]: Remind me why we aren't doing this for DOUBLE too?

Comment (arittr) [reply src/process-columnar-results.js]: DOUBLE already matches the mapdql output from what I can tell

**Comment by tmostak** [comment]
Hold on, I thought we said in the Jira issue that we didn't want to reduce precision going into Immerse/charting, as we need all the precision we can get to ensure that particularly pointmap/scatterplot don't end up gridded (rounding a few digits can make this happen easily as far as I remember).

**Comment by arittr** [comment]
retest this please

**Pull request merged**